homeassistant/components/http/auth.py

```diff
@@ -105,11 +105,11 @@ def async_user_not_allowed_do_auth(
             return "User is local only"
 
     try:
-        remote = ip_address(request.remote)  # type: ignore[arg-type]
+        remote_address = ip_address(request.remote)  # type: ignore[arg-type]
```

Comment:
This seems out of place, why is this here?

Reply:
It's there because the `remote` was used for two completely different purposes.
Before bumping `hass-nabucasa`, mypy did not complain because `hass-nabucasa` was untyped.

Reply:
ref https://github.com/home-assistant/core/pull/92484/files#diff-6e7a529a12ddb86424b2771a1c8fefe60b3b2335265bae1de5cb78a64177c5a3R102